docs/MO_DG/prepare_model/convert_model/supported_model_formats.md

```diff
@@ -16,32 +16,533 @@
+
+              .. dropdown:: List of supported formats:
+
+                 * **Python objects**:
+
+                   * ``torch.nn.Module``
+                   * ``torch.jit.ScriptModule``
+                   * ``torch.jit.ScriptFunction``
+
+              .. code-block:: py
+                 :force:
+
+                 model = torchvision.models.resnet50(pretrained=True)
+                 ov_model = convert_model(model)
+                 compiled_model = core.compile_model(ov_model, "AUTO")
+
+              For more details on conversion, refer to the 
+              :doc:`guide <openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_PyTorch>` 
+              and an example `tutorial <https://docs.openvino.ai/nightly/notebooks/102-pytorch-onnx-to-openvino-with-output.html>`__ 
+              on this topic.
+
+   .. tab-item:: TensorFlow
+      :sync: tf
+
+      .. tab-set::
+
+         .. tab-item:: Python
+            :sync: py
+
+            * The ``convert_model()`` method:
+
+              When you use the ``convert_model()`` method, you have more control and you can specify additional adjustments for ``ov.Model``. The ``read_model()`` and ``compile_model()`` methods are easier to use, however, they do not have such capabilities. With ``ov.Model`` you can choose to optimize, compile and run inference on it or serialize it into a file for subsequent use.
+
+              .. dropdown:: List of supported formats:
+
+                 * **Files**:
+
+                   * SavedModel - ``<SAVED_MODEL_DIRECTORY>`` or ``<INPUT_MODEL>.pb``
+                   * Checkpoint - ``<INFERENCE_GRAPH>.pb`` or ``<INFERENCE_GRAPH>.pbtxt``
+                   * MetaGraph - ``<INPUT_META_GRAPH>.meta``
+
+                 * **Python objects**:
+
+                   * ``tf.keras.Model``
+                   * ``tf.keras.layers.Layer``
+                   * ``tf.Module``
+                   * ``tf.compat.v1.Graph``
+                   * ``tf.compat.v1.GraphDef``
+                   * ``tf.function``
+                   * ``tf.compat.v1.session``
+                   * ``tf.train.checkpoint``
+
+              .. code-block:: py
+                 :force:
+
+                 ov_model = convert_model("saved_model.pb")
+                 compiled_model = core.compile_model(ov_model, "AUTO")
+
+              For more details on conversion, refer to the 
+              :doc:`guide <openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_TensorFlow>` 
+              and an example `tutorial <https://docs.openvino.ai/nightly/notebooks/101-tensorflow-to-openvino-with-output.html>`__ 
+              on this topic. 
+
+            * The ``read_model()`` and ``compile_model()`` methods:
```

Comment:
I propose to remove `read_model` and `compile_model` methods from here at all because this part documentation should be about `convert_model`. `read_model` and `compile_model` have own documentation

Reply:
`read_model` and `compile_model` overloads this documentation now.

Reply:
Could we keep convert_model and compile_model together? It would meet the requirements set out in CVS-113150

Reply:
Let leave it as you suggested.